src/jsifier.js

```diff
@@ -130,11 +140,7 @@ function JSify(data, functionsOnly) {
       }
 
       // $ident's are special, we do not prefix them with a '_'.
```

Comment:
This comment doesn't make sense here anymore.  It can either move up to mangleCSymbolName or be removed I guess?

Reply:
check